influxql/ast.go

```diff
@@ -555,7 +592,7 @@ func (s *CreateRetentionPolicyStatement) String() string {
 
 // RequiredPrivileges returns the privilege required to execute a CreateRetentionPolicyStatement.
 func (s *CreateRetentionPolicyStatement) RequiredPrivileges() ExecutionPrivileges {
-	return ExecutionPrivileges{{Name: "", Privilege: AllPrivileges}}
+	return ExecutionPrivileges{{Admin: true, Name: "", Privilege: AllPrivileges}}
```

Comment:
I made any statement that previously required `AllPrivileges` a statement that requires admin access. My assumption was that no additional privileges should be granted for `AllPrivileges` that were not already allowed by the `Read` and `Write` privileges. Therefore, all statements that were previously `AllPrivileges` now require admin privilege.

RP creation seems like something that _could_ only require the `Write` privilege, since there should be no way to delete data by creating a new RP.